analyses/org.osate.analysis.flows/src/org/osate/analysis/flows/FlowLatencyAnalysisSwitch.java

```diff
@@ -835,41 +974,6 @@ public AnalysisResult invoke(ComponentInstance ci, SystemOperationMode som, bool
 		return results;
 	}
 
-	public AnalysisResult invoke(EndToEndFlowInstance etef) {
-		return invoke(etef, null, true, true, true, true);
-	}
-
-	/**
-	 * Invoke the analysis on all ETEF owned by the given component instance and return Result collection
-	 *
-	 * @param etef The end to end flow instance
-	 * @param som The mode to run the analysis in. If null then run all SOMs
-	 * @param asynchronousSystem Whether the system is treated as asynchronous
-	 * @param majorFrameDelay Whether partition output is performed at a major frame (as opposed to the partition end)
-	 * @param worstCaseDeadline Use deadline based processing (as opposed to max compute execution time)
-	 * @param bestCaseEmptyQueue Assume empty queue (instead of full)
-	 * @return A populated report in AnalysisResult format.
-	 */
-	public AnalysisResult invoke(EndToEndFlowInstance etef, SystemOperationMode som, boolean asynchronousSystem,
```

Comment:
Why did you remove this method? It could be used by a 3rd party plugin.

Reply:
I put it back